ERROR - 2023-03-30 06:16:21 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\SIPIUTANG\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\laragon\www\SIPIUTANG\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'db_piutang', 3306, '', 0)
#1 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\SIPIUTANG\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `tb_invo...', Array, false)
#4 C:\laragon\www\SIPIUTANG\app\Models\ModelInvoice.php(29): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\SIPIUTANG\app\Controllers\AJAX\Invoice.php(13): App\Models\ModelInvoice->getInvoice(NULL)
#6 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(934): App\Controllers\AJAX\Invoice->index()
#7 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\Invoice))
#8 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\SIPIUTANG\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\SIPIUTANG\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\SIPIUTANG\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\SIPIUTANG\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `tb_invo...', Array, false)
#3 C:\laragon\www\SIPIUTANG\app\Models\ModelInvoice.php(29): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\SIPIUTANG\app\Controllers\AJAX\Invoice.php(13): App\Models\ModelInvoice->getInvoice(NULL)
#5 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(934): App\Controllers\AJAX\Invoice->index()
#6 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\Invoice))
#7 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\SIPIUTANG\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\SIPIUTANG\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#10 {main}
CRITICAL - 2023-03-30 06:16:21 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `tb_invoice`.`id`, `no_invoice`, `tgl_invoice`, `termin`, `jth_tmpo`, `nilai_invoice`, `sisa_hutang`, `id_customer`, `id_user`, `nama_cstmr`, `id_bank`, `rekening`, `atas_nama`, `nama_usaha`, `id_customer`, `id_invoice_ekspedisi`, `no_invoice_ekspedisi`, `tgl_pengiriman`, `nama_ekspedisi`, `no_telepon_ekspedisi`
FROM `tb_invoice`
JOIN `tb_customer` ON `tb_customer`.`id` = `tb_invoice`.`id_customer`
LEFT JOIN `tb_invoice_ekspedisi` ON `tb_invoice_ekspedisi`.`id` = `tb_invoice`.`id_invoice_ekspedisi`
LEFT JOIN `tb_ekspedisi` ON `tb_ekspedisi`.`id` = `tb_invoice_ekspedisi`.`id_ekspedisi`', [], false)
 3 APPPATH\Models\ModelInvoice.php(29): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\AJAX\Invoice.php(13): App\Models\ModelInvoice->getInvoice(null)
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\Invoice->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\Invoice))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-03-30 06:16:25 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\SIPIUTANG\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\laragon\www\SIPIUTANG\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'db_piutang', 3306, '', 0)
#1 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\SIPIUTANG\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\SIPIUTANG\app\Models\ModelCustomer.php(16): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\SIPIUTANG\app\Controllers\AJAX\Customer.php(13): App\Models\ModelCustomer->getCustomer(NULL)
#6 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(934): App\Controllers\AJAX\Customer->index()
#7 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\Customer))
#8 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\SIPIUTANG\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\SIPIUTANG\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\SIPIUTANG\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\SIPIUTANG\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\SIPIUTANG\app\Models\ModelCustomer.php(16): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\SIPIUTANG\app\Controllers\AJAX\Customer.php(13): App\Models\ModelCustomer->getCustomer(NULL)
#5 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(934): App\Controllers\AJAX\Customer->index()
#6 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\Customer))
#7 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\SIPIUTANG\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\SIPIUTANG\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#10 {main}
CRITICAL - 2023-03-30 06:16:25 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_customer`', [], false)
 3 APPPATH\Models\ModelCustomer.php(16): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\AJAX\Customer.php(13): App\Models\ModelCustomer->getCustomer(null)
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\Customer->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\Customer))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-03-30 06:16:29 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\laragon\www\SIPIUTANG\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\laragon\www\SIPIUTANG\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'db_piutang', 3306, '', 0)
#1 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\SIPIUTANG\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `tb_invo...', Array, false)
#4 C:\laragon\www\SIPIUTANG\app\Models\ModelInvoiceEkspedisi.php(17): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\SIPIUTANG\app\Controllers\AJAX\InvoiceEkspedisi.php(13): App\Models\ModelInvoiceEkspedisi->getInvoiceEkspedisi(NULL)
#6 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->index()
#7 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
#8 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\SIPIUTANG\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\SIPIUTANG\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\laragon\www\SIPIUTANG\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\SIPIUTANG\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `tb_invo...', Array, false)
#3 C:\laragon\www\SIPIUTANG\app\Models\ModelInvoiceEkspedisi.php(17): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\SIPIUTANG\app\Controllers\AJAX\InvoiceEkspedisi.php(13): App\Models\ModelInvoiceEkspedisi->getInvoiceEkspedisi(NULL)
#5 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->index()
#6 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
#7 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\laragon\www\SIPIUTANG\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\laragon\www\SIPIUTANG\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#10 {main}
CRITICAL - 2023-03-30 06:16:29 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `tb_invoice_ekspedisi`.`id`, `no_invoice_ekspedisi`, `banyak_barang`, `nilai_invoice_ekspedisi`, `tgl_pengiriman`, `keterangan`, `sisa_hutang_ekspedisi`, `id_ekspedisi`, `nama_ekspedisi`, `no_telepon_ekspedisi`
FROM `tb_invoice_ekspedisi`
JOIN `tb_ekspedisi` ON `tb_ekspedisi`.`id` = `tb_invoice_ekspedisi`.`id_ekspedisi`', [], false)
 3 APPPATH\Models\ModelInvoiceEkspedisi.php(17): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(13): App\Models\ModelInvoiceEkspedisi->getInvoiceEkspedisi(null)
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 07:12:45 --> Undefined array key 0
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 42.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(42): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 42)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 07:13:08 --> Undefined array key 0
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 42.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(42): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 42)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 07:15:19 --> Attempt to read property "dataInvoiceTable" on array
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 45.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(45): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "dataInvoiceTable" on array', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 45)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 07:16:38 --> Attempt to read property "dataInvoiceTable" on array
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 45.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(45): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "dataInvoiceTable" on array', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 45)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 07:16:40 --> Attempt to read property "dataInvoiceTable" on array
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 45.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(45): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "dataInvoiceTable" on array', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 45)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 07:17:10 --> Array to string conversion
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 45.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(45): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 45)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 07:17:34 --> Undefined array key "dataInvoiceTable"
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 45.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(45): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "dataInvoiceTable"', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 45)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 07:18:20 --> Attempt to read property "dataInvoiceTable" on array
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 45.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(45): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "dataInvoiceTable" on array', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 45)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 07:18:45 --> Attempt to read property "dataInvoiceTable" on array
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 45.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(45): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "dataInvoiceTable" on array', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 45)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 07:19:14 --> Attempt to read property "dataInvoiceTable" on array
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 45.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(45): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "dataInvoiceTable" on array', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 45)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 07:19:49 --> Undefined array key 0
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 41.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(41): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 41)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 07:21:21 --> Attempt to read property "dataInvoiceTable" on array
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 45.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(45): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "dataInvoiceTable" on array', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 45)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 07:21:32 --> Attempt to read property "dataInvoiceTable" on array
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 45.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(45): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "dataInvoiceTable" on array', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 45)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 07:23:41 --> Attempt to read property "dataInvoiceTable" on array
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 49.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(49): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "dataInvoiceTable" on array', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 49)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-03-30 08:02:46 --> mysqli_sql_exception: Unknown column 'sisa_hutang' in 'field list' in C:\laragon\www\SIPIUTANG\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\laragon\www\SIPIUTANG\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\laragon\www\SIPIUTANG\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\laragon\www\SIPIUTANG\app\Models\ModelInvoiceEkspedisi.php(35): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\laragon\www\SIPIUTANG\app\Controllers\AJAX\InvoiceEkspedisi.php(58): App\Models\ModelInvoiceEkspedisi->insertData(Array)
#6 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
#7 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
#8 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\SIPIUTANG\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\SIPIUTANG\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#11 {main}
CRITICAL - 2023-03-30 08:02:46 --> Unknown column 'sisa_hutang' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb_invoice_ekspedisi` (`no_invoice_ekspedisi`, `nilai_invoice_ekspedisi`, `banyak_barang`, `keterangan`, `id_ekspedisi`, `sisa_hutang`) VALUES (:no_invoice_ekspedisi:, :nilai_invoice_ekspedisi:, :banyak_barang:, :keterangan:, :id_ekspedisi:, :sisa_hutang:)', [...], false)
 2 APPPATH\Models\ModelInvoiceEkspedisi.php(35): CodeIgniter\Database\BaseBuilder->insert([...])
 3 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(58): App\Models\ModelInvoiceEkspedisi->insertData([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-03-30 08:09:14 --> mysqli_sql_exception: Unknown column 'tgl_pengiriman' in 'field list' in C:\laragon\www\SIPIUTANG\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\laragon\www\SIPIUTANG\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT `tb_invo...', 0)
#1 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `tb_invo...')
#2 C:\laragon\www\SIPIUTANG\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `tb_invo...')
#3 C:\laragon\www\SIPIUTANG\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `tb_invo...', Array, false)
#4 C:\laragon\www\SIPIUTANG\app\Models\ModelInvoice.php(29): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\SIPIUTANG\app\Controllers\AJAX\Invoice.php(13): App\Models\ModelInvoice->getInvoice(NULL)
#6 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(934): App\Controllers\AJAX\Invoice->index()
#7 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\Invoice))
#8 C:\laragon\www\SIPIUTANG\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\SIPIUTANG\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\laragon\www\SIPIUTANG\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#11 {main}
CRITICAL - 2023-03-30 08:09:14 --> Unknown column 'tgl_pengiriman' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `tb_invoice`.`id`, `no_invoice`, `tgl_invoice`, `termin`, `jth_tmpo`, `nilai_invoice`, `sisa_hutang`, `id_customer`, `id_user`, `nama_cstmr`, `id_bank`, `rekening`, `atas_nama`, `nama_usaha`, `id_customer`, `id_invoice_ekspedisi`, `no_invoice_ekspedisi`, `tgl_pengiriman`, `nama_ekspedisi`, `no_telepon_ekspedisi`
FROM `tb_invoice`
JOIN `tb_customer` ON `tb_customer`.`id` = `tb_invoice`.`id_customer`
LEFT JOIN `tb_invoice_ekspedisi` ON `tb_invoice_ekspedisi`.`id` = `tb_invoice`.`id_invoice_ekspedisi`
LEFT JOIN `tb_ekspedisi` ON `tb_ekspedisi`.`id` = `tb_invoice_ekspedisi`.`id_ekspedisi`', [], false)
 2 APPPATH\Models\ModelInvoice.php(29): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\AJAX\Invoice.php(13): App\Models\ModelInvoice->getInvoice(null)
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\Invoice->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\Invoice))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 08:15:25 --> Undefined array key "id_invoice"
in APPPATH\Controllers\AJAX\InvoiceEkspedisi.php on line 65.
 1 APPPATH\Controllers\AJAX\InvoiceEkspedisi.php(65): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_invoice"', 'APPPATH\\Controllers\\AJAX\\InvoiceEkspedisi.php', 65)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\InvoiceEkspedisi->insertData()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\InvoiceEkspedisi))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 15:06:08 --> Undefined index: password
in APPPATH\Controllers\AJAX\User.php on line 74.
 1 APPPATH\Controllers\AJAX\User.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index: password', 'APPPATH\\Controllers\\AJAX\\User.php', 74, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\User->updateData([...])
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\User))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 15:35:11 --> You must set the database table to be used with your query.
in SYSTEMPATH\Database\BaseConnection.php on line 882.
 1 SYSTEMPATH\Model.php(642): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH\Model.php(832): CodeIgniter\Model->builder()
 3 APPPATH\Controllers\AJAX\Customer.php(53): CodeIgniter\Model->__call('getUser', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\Customer->getDetail([...])
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\Customer))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 16:14:12 --> Undefined index: username
in APPPATH\Controllers\AJAX\User.php on line 77.
 1 APPPATH\Controllers\AJAX\User.php(77): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index: username', 'APPPATH\\Controllers\\AJAX\\User.php', 77, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\User->updateData([...])
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\User))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 16:14:33 --> Undefined index: password
in APPPATH\Controllers\AJAX\User.php on line 74.
 1 APPPATH\Controllers\AJAX\User.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index: password', 'APPPATH\\Controllers\\AJAX\\User.php', 74, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\User->updateData([...])
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\User))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 16:15:00 --> Undefined index: password
in APPPATH\Controllers\AJAX\Customer.php on line 66.
 1 APPPATH\Controllers\AJAX\Customer.php(66): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index: password', 'APPPATH\\Controllers\\AJAX\\Customer.php', 66, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\Customer->updateData([...])
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\Customer))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 16:16:52 --> Undefined variable: datatext
in APPPATH\Controllers\AJAX\Customer.php on line 65.
 1 APPPATH\Controllers\AJAX\Customer.php(65): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: datatext', 'APPPATH\\Controllers\\AJAX\\Customer.php', 65, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\Customer->updateData([...])
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\Customer))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-30 16:19:19 --> Undefined variable: datatext
in APPPATH\Controllers\AJAX\Customer.php on line 67.
 1 APPPATH\Controllers\AJAX\Customer.php(67): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: datatext', 'APPPATH\\Controllers\\AJAX\\Customer.php', 67, [...])
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AJAX\Customer->updateData([...])
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AJAX\Customer))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
